file_formats/ie_formats/itm_v1.htm

```diff
@@ -1228,7 +1228,7 @@
           <td>Damage strength bonus (BGEE split bit 0)</td>
           <td><b><a name="ExtHeader_Flags_BIT10">Hostile</a></b></td>
           <td><span class="unknown">Unknown</span></td>
-          <td>EE/ToBEx: Can target invisible</td>
+          <td>EE/ToBEx: Cannot target invisible</td>
```

Comment:
nice catch, it's indeed the reverse of the one in SPL.

Reply:
Yeah, it's the reverse... In NearInfinity it's already labeled correctly, I just forgot to reverse it when I copied/pasted from `SPL`...